assets/js/circles.js

```diff
@@ -166,17 +50,13 @@ function generateRandomCircle() {
 }
 
 function generateRandomPoint() {
```

Comment:
tutaj jeszcze mozna w sumie to CANVAS przekazywac jako argument funkcji, ale w tym przypadku to malo istotne